src/filters/filters-menu.js

```diff
@@ -90,20 +92,38 @@ class TableFiltersMenuView {
     this.updateMenu(view);
   }
 
-  onOpen() {
-    ReactDOM.render(
-      <TableFiltersComponent
-        dom={this.tablesData.dom}
-        headerPos={this.tablesData.headerPos}
-        pos={this.tablesData.pos}
-        table={this.tablesData.node}
-        view={this.view}
-      />,
-      this.popUpDOM
-    );
+  onOpen(manage = false) {
```

Comment:
do we want this to be an enum or string instead of a boolean? in case we add more states